packages/trader/src/Modules/MT5/Components/mt5-personal-details-form.jsx

```diff
@@ -55,11 +55,17 @@ export const InputField = ({ name, optional = false, ...props }) => (
     </Field>
 );
 
-const validatePersonalDetails = (values, residence_list, account_opening_reason) => {
+const validatePersonalDetails = (values, residence_list, account_opening_reason, landing_company) => {
+    const is_tin_required = landing_company?.config.tax_details_required || false;
+    const tin_regex = landing_company?.config?.tin_format?.[0];
```

Comment:
```suggestion
    const tin_regex = landing_company?.config?.tin_format?.[0] ?? true;
```

on line 67 saw added fallback for this regex, i guess we can put the fallback here instead

Reply:
i guess we can revert it back, my bad, it was correct, line 67 you were checking tin_regex, whether its true or false, if its like this then it will always resolve to true